settings.py

```diff
@@ -458,6 +458,7 @@ def get_locales():
     'dbgettext',
     'honeypot',
     'cacheback',
+    'django_extensions',
```

Comment:
It adds a few useful management commands, and provides a model field for easy timespamps. It's well maintained and really worth the addition. We previously had it in settings_local.py but there was no reason to not have it here either.